src/components/Nav/index.js

```diff
@@ -0,0 +1,89 @@
+import React, {useState} from 'react';
+import styled, {css} from 'styled-components';
+import {FaAlignJustify, FaBackspace} from 'react-icons/fa';
+
+function Nav() {
+    const [showMenu, setShowMenu] = useState(false);
+
+    const toggleMenu = () => {
+        setShowMenu(!showMenu);
+    };
+
+    return (
+        <div>
+            <HeaderNav>
+                <div className="menu-icon" id="open">
+                    <button
+                        data-testid="navButton"
+                        aria-label="navButton"
```

Comment:
This is not a good `aria-label`. The aria-label should be descriptive and in real English/German etc. 
```suggestion
                        aria-label="Open navigation"
```

Reply:
i did it